src/components/Person/Person.jsx

```diff
@@ -1 +1,29 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+
+      {age ? (
+        <p className="Person__age">{`I am ${age}`}</p>) : ''}
+
+      {isMarried ? (
+        <p className="Person__partner">
+          {`${partnerName} is my ${sex === 'm'
+            ? 'wife'
+            : 'husband'
+          }`}
```

Comment:
Try to avoid ternary nesting. Move this expression to variable 
After fixing avoid tag repeats `<p className="Person__partner"> .... </p>`. Try use ternary inside this tag